tests/components/solarlog/snapshots/test_diagnostics.ambr

```diff
@@ -0,0 +1,64 @@
+# serializer version: 1
+# name: test_entry_diagnostics
+  dict({
+    'config_entry': dict({
+      'data': dict({
+        'extended_data': True,
+        'host': '**REDACTED**',
+        'name': 'Solarlog test 1 2 3',
+      }),
+      'disabled_by': None,
+      'domain': 'solarlog',
+      'entry_id': 'ce5f5431554d101905d31797e1232da8',
```

Comment:
Provide a fixed `entry_id` or exclude it

Reply:
What is preferred alternative?

Reply:
Just looked it up and noticed that in conftest.py on L42 entry_id is already set.